src/phoenix/datasets/types.py

```diff
@@ -80,6 +82,20 @@ class Dataset:
     version_id: DatasetVersionId
     examples: Sequence[Example]
 
+    @cached_property
+    def dataframe(self) -> pd.DataFrame:
```

Comment:
Non-blocking. Wonder if "as_dataframe" would be more appropriate? https://www.tensorflow.org/datasets/api_docs/python/tfds/as_dataframe

Reply:
sg. i'll rename